tests/unit/test_io.py

```diff
@@ -296,3 +298,62 @@ def test_parquet_lists(tmpdir, freq_threshold):
     df_out = cudf.read_parquet(out_paths)
 
     assert df_out["Authors"].to_arrow().to_pylist() == [[1], [1, 4], [2, 3], [3]]
+
+
+@pytest.mark.parametrize("part_size", [None, "1KB"])
+@pytest.mark.parametrize("size", [100, 1000])
+@pytest.mark.parametrize("nfiles", [1, 2])
+def test_avro_basic(tmpdir, part_size, size, nfiles):
+
+    # Require uavro and fastavro library.
+    # Note that fastavro is only required to write
+    # avro files for testing, while uavro is actually
+    # used by AvroDatasetEngine.
+    fa = pytest.importorskip("fastavro")
```

Comment:
We should probably get fastavro/uavro installed on the CI container to test this out (more a note for myself than anything =)

Reply:
nvm - I see alberto has done this. (thanks!)